### ISSUE TITLE
Travis CI builds failing from broken link

### ISSUE THREAD

**Issue by kevjin** [freeCodeCamp/how-to-contribute-to-open-source#312]
All Travis CI builds are failing [and have been since March 21st](https://github.com/freeCodeCamp/how-to-contribute-to-open-source/pulls?q=is%3Apr+is%3Aclosed). This has happened before, see #177, although the root cause might be different, I haven't investigated it.

**Comment by erictleung** [comment]
Nice catch.

The problem is that this link

```
https://pybee.org/contributing/how/first-time/
```

is broken in the `PROJECTS.md` file and needs to be replaced with

```
https://beeware.org/contributing/how/first-time/
```

This is up for grabs for anyone who wants to take this on. Just be sure to reference this issue for whoever picks this up by adding `Closes #312` in your pull request message.

**Comment by kevjin** [comment]
Just opened a fix above, thanks!
